efficientdet/keras/efficientdet_keras.py

```diff
@@ -569,6 +567,70 @@ def call(self, inputs, training, **kwargs):
     return box_outputs
 
 
+class SegmentationHead(tf.keras.layers.Layer):
```

Comment:
One more comment: would be great if you could add a test for the segmentationHead and config.head=['object_detection', 'segmentation']